articles/puppet-module.md

```diff
@@ -99,11 +99,11 @@ Set the `ensure` parameter to `absent` to create teams that exclude specific pro
 
 For more examples check out the `examples/` folder in Fleet's GitHub repository [here](https://github.com/fleetdm/fleet/tree/main/ee/tools/puppet/fleetdm/examples).
 
-> Note that all teams created by Puppet inherit the bootstrap package, macOS Setup Assistant settings, and end user authentication settings from "No team." Learn more about these [here](./mdm-setup.md). In addition all teams automatically enable disk encryption. Learn more about disk encryption [here](./MDM-disk-encryption.md).
+> Note that all teams created by Puppet inherit the bootstrap package, macOS Setup Assistant settings, and end user authentication settings from "No team." Learn more about these [here](https://fleetdm.com/guides/macos-setup-experience). In addition all teams automatically enable disk encryption. Learn more about disk encryption [here](https://fleetdm.com/guides/enforce-disk-encryption).
 
 ## Release host
 
-If you set `await_device_configured` to `true` in your [macOS Setup Assistant settings](./mdm-setup.md#macos-setup-assistant), you can use the `fleetdm::release_device` function to release the host from the Setup Assistant. 
+If you set `enable_release_device_manually` to `true` in your [macOS setup experience](https://fleetdm.com/docs/rest-api/rest-api#configure-setup-experience), you can use the `fleetdm::release_device` function to release the host from the Setup Assistant. 
```

Comment:
The `await_device_configured` setting in Setup Assistant was effectively overruled by Fleet with the addition of `enable_release_device_manually`  in [4.58](https://github.com/fleetdm/fleet/blob/76e1862bf69be8ad299b8e0e599293f4344bc888/articles/fleet-4.48.0.md#L47-L48), but it seems the docs weren't updated. I took my best shot, but I wasn't closely involved in the development so it would be good for someone from product to weigh in here.

Reply:
^^ @noahtalerman @marko-lisica

Reply:
Looks good to me. [This is](https://www.figma.com/design/vFXKF3NwTBLmnzzk4KrrKl/%239147-Pre-fill-and-lock-local-account-creation-screen-during-out-of-the-box-macOS-setup?node-id=356-760&node-type=instance&t=PfDzDX5kVRRpoAZQ-11) how the new behavior looks like. Fleet adds `await_device_configured` by default and won't let the user to change that.